.circleci/config.yml

```diff
@@ -251,6 +261,10 @@ workflows:
       - build-containers:
           requires:
             - test-sanity-check
+# cleanup
+      - cleanup-images:
+          requires:
+            - test-sanity-check
```

Comment:
Why we can't just wait here integration-test job?

Reply:
1) in such case the job will be run only if testing passes, which is not that we want
2) "docker-push-tag" job needs the images too

Reply:
Is `circle ci` has not solutions for problems 1. 2 ?

Reply:
I haven't found any way. 
What we need is to run the cleanup job after all of the (build, test, push) jobs are finished (if started), regardless of their exit statuses.